src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -269,6 +269,7 @@
 	<bean id="preferencesDao" parent="preferencesDao-parentBean"/>
 	<bean id="preferencesDao-parentBean" abstract="true" class="${kfs.documentstore.class}">
 		<property name="documentStoreClient" ref="documentStoreClient" />
+		<property name="tableNamePrefix" value="${kfs.documentstore.tablename.prefix}" />
```

Comment:
The default interface methods didn't work well with Spring and BeanUtils. Instead of trying to figure that out (if even possible), I just added the new field, setter and getter to CuPreferencesDaoMongo.